idl/storage_service.idl.hh

```diff
@@ -32,6 +32,7 @@ struct raft_topology_cmd {
         fence
```

Comment:
Maybe https://github.com/scylladb/scylladb/issues/16644 is the issue.

After our discussion with @gleb-cloudius -- one approach that we both like is to not update system.peers inside topology_state_load, but inside a separate subscriber. (That subscribes to both gossiper and topology_state_load calls.)

This could be the same subscriber which updates raft_address_map. And the raft_address_map update would happen after system.peers persisting. Therefore, `wait_for_ip` finishing would guarantee that the entry was persisted.

Reply:
mentioned this in the [todo list](https://github.com/scylladb/scylladb/pull/16639#issuecomment-1876567388)